index.css

```diff
@@ -12,12 +12,74 @@ a {
   color: inherit;
 }
 
-body {
-  padding: 0;
-  margin: 0;
+/* Popup window */
+.default {
+  display: none;
+}
+
+.popup-header {
+  display: flex;
+  justify-items: baseline;
+  margin-top: 16px;
+  margin-bottom: 40px;
+}
+
+.title-style {
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 24px;
+  line-height: 44px;
+  color: #172b4d;
+  width: 200px;
+  margin-left: 18px;
+}
+
+#projectDiscription {
+  margin-top: 10px;
+  margin-bottom: 10px;
+  font-family: "Inter", Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
 }
 
+#infoButton {
+  display: flex;
+  margin-left: 0;
+  margin-bottom: 20px;
+}
+
+.icon-info {
+  margin-left: 20px;
+}
+
+.popup {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  position: fixed;
+  overflow: scroll;
+  width: 343px;
+  height: 770px;
+  left: calc(50% - 343px / 2);
+  top: 40px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+}
```

Comment:
- [ ] As required by the previous reviewer asked, In the mobile version: The pop-up is not scrollable. Please you can just remove the padding-top and the button will appear or make it scrollable.

![image](https://user-images.githubusercontent.com/35535130/157770022-589a148b-6400-41fb-8763-90dcdb36bd7c.png)